mmdet/models/task_modules/prior_generators/utils.py

```diff
@@ -39,15 +42,18 @@ def anchor_inside_flags(flat_anchors,
     return inside_flags
 
 
-def calc_region(bbox, ratio, featmap_size=None):
+def calc_region(bbox: Tensor,
+                ratio: float,
+                featmap_size: Optional[Tuple] = None) -> Tuple[int]:
     """Calculate a proportional bbox region.
 
     The bbox center are fixed and the new h' and w' is h * ratio and w * ratio.
 
     Args:
         bbox (Tensor): Bboxes to calculate regions, shape (n, 4).
         ratio (float): Ratio of the output region.
-        featmap_size (tuple): Feature map size used for clipping the boundary.
+        featmap_size (tuple, Optional): Feature map size used for
+            clipping the boundary. Defaults to None.
```

Comment:
It would be better if we indicate it the featmap_size is in (height, width) order

Reply:
Thank u for reminding.